### PR TITLE
Feature/translation pt 2

### PR DESCRIPTION
**Pull request by luispinho**
More Portuguese translations.

This the second of three Pull Requests. Please check out and merge [the first PR](https://github.com/ghostfolio/ghostfolio/pull/2073) **before merging this one**, as this is based on that one.

### CODE REVIEW

**Comment by dtslvr** [comment]
> Please check out and merge [the first PR](https://github.com/ghostfolio/ghostfolio/pull/2073) **before merging this one**, as this is based on that one.

Ok, the first PR has been merged. Can you please resolve the conflicts @luispinho here?

**Comment by luispinho** [comment]
> > Please check out and merge [the first PR](https://github.com/ghostfolio/ghostfolio/pull/2073) **before merging this one**, as this is based on that one.
> 
> Ok, the first PR has been merged. Can you please resolve the conflicts @luispinho here?

Fixed 👍

File: apps/client/src/locales/messages.pt.xlf [hunk 1]
```diff
@@ -815,15 +815,15 @@
       </trans-unit>
       <trans-unit id="a5e368dd1104a015e020de33a532e53e4d97525f" datatype="html">
         <source> Last Request </source>
-        <target state="new"> Last Request </target>
+        <target state="translated"> Último Pedido</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:818]: Missing space at the end (I think it does not matter)

File: apps/client/src/locales/messages.pt.xlf [hunk 2]
```diff
@@ -2751,119 +2751,119 @@
...
+        <target state="translated">Conta</target>
         <context-group purpose="location">
           <context context-type="sourcefile">libs/ui/src/lib/i18n.ts</context>
           <context context-type="linenumber">4</context>
         </context-group>
       </trans-unit>
       <trans-unit id="4574987680940794089" datatype="html">
         <source>Asset Class</source>
-        <target state="new">Asset Class</target>
+        <target state="new">Classe de Ativo</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:2850]: state="translated"

File: apps/client/src/locales/messages.pt.xlf [hunk 3]
```diff
@@ -3567,55 +3567,55 @@
...
+        <target state="translated">Suporte por Email e Chat</target>
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/pages/pricing/pricing-page.html</context>
           <context context-type="linenumber">314</context>
         </context-group>
       </trans-unit>
       <trans-unit id="3298117765569632011" datatype="html">
         <source>Switch to Ghostfolio Premium or Ghostfolio Open Source easily</source>
-        <target state="new">Switch to Ghostfolio Premium or Ghostfolio Open Source easily</target>
+        <target state="translated">Mude para o Ghostfolio Premiumou Ghostfolio Open Source  facilmente<</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:3594]: Premiumou? I have also seen Premium

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thank you @luispinho. I have added some comments.

File: apps/client/src/locales/messages.pt.xlf (hunk 3 above)
Comment (luispinho) [reply apps/client/src/locales/messages.pt.xlf:3594]: Missed a space between "Premium" and "ou", fixed

**Comment by luispinho** [comment]
PR remarks have been fixed 👍

**Review by dtslvr — APPROVED** [review]
Thanks for the changes @luispinho!

**Pull request merged**